torch/quantization/fx/backend_config_dict/fbgemm.py

```diff
@@ -0,0 +1,12 @@
+import torch
+from ..quantization_patterns import *
+from ..pattern_utils import get_default_quant_patterns
+
+def get_fbgemm_backend_config_dict():
+    """ Get the backend config dictionary for fbgemm backend
+    NOTE: Current api is not final, it's just to unblock experimentation for new backends
```

Comment:
can we also add some checking that the values are valid, or add a TODO to do it before users start using the API?

Reply:
you mean `validate_backend_config_dict`? yeah we have that in __init__.py